.php-cs-fixer.dist.php

```diff
@@ -12,15 +12,15 @@
  * with this source code in the file LICENSE.
  */
 
```

Comment:
> > if we should apply `PHP` and other specific markers where applicable (instead of `EOD` or other default marker)
> 
> we can apply `EOD` now to prevent conflicts, as this is pure internal CS change, `PHP` can be added later if wanted